src/ui/EastRegion.js

```diff
@@ -487,8 +487,11 @@ EastRegion = function(c) {
                         listeners: {
                             'render': function(label) {
                                 label.getEl().on('click', function() {
-                                    numberOfCommentsToDisplay += 3;
-                                    this.up('#interpretationPanel' + interpretation.id).updateInterpretationPanelItems();
+                                    numberOfCommentsToDisplay = comments.length;
+                                    var panel = this.up('#interpretationPanel' + interpretation.id);
```

Comment:
Is it not enough to call updateInterpretationPanelItems setting the numberOfCommentsToDisplay to the length of the comments as we used to do?

Reply:
Yes, but  `updateInterpretationPanelItems` redraws the whole component, it was visually noticeable.